lib/apis/v2/globalnetworkpolicy.go

```diff
@@ -49,7 +49,73 @@ type GlobalNetworkPolicy struct {
 	// Standard object's metadata.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	// Specification of the Policy.
-	Spec PolicySpec `json:"spec,omitempty"`
+	Spec GlobalNetworkPolicySpec `json:"spec,omitempty"`
+}
+
+type GlobalNetworkPolicySpec struct {
+	// Order is an optional field that specifies the order in which the policy is applied.
+	// Policies with higher "order" are applied after those with lower
+	// order.  If the order is omitted, it may be considered to be "infinite" - i.e. the
+	// policy will be applied last.  Policies with identical order will be applied in
+	// alphanumerical order based on the Policy "Name".
+	Order *float64 `json:"order,omitempty"`
```

Comment:
Kind of a nit, but could you put newlines between the field and the next set of comments?